source-map-uploader-plugin.js

```diff
@@ -44,8 +44,17 @@ class BugsnagSourceMapUploaderPlugin {
           // for each *.map file, find a corresponding source file in the chunk
           const source = map ? chunk.files.find(file => file === map.replace('.map', '')) : null
 
-          if (!source || !map) {
-            console.warn(`${LOG_PREFIX} no source/map pair found for chunk "${chunk.id}"`)
+          if (!source) {
+            console.warn(`${LOG_PREFIX} no corresponding source found for "${map}" in chunk "${chunk.id}"`)
+            return null
+          }
+
+          if (!compilation.assets[source]) {
```

Comment:
If this doesn't exist, it looks like it's still possible for `existsAt` to be called on it at line 62.  Will this cause another issue?

Reply:
Good catch, thanks! I'm missing a `return null` here.